From d20047c9b4805583904b87774008ff914484f2dd Mon Sep 17 00:00:00 2001
From: vijayabharathi shetty <vijayabharathix.shetty@intel.com>
Date: Fri, 9 Jun 2023 16:28:05 +0000
Subject: [PATCH] CTL: Add rpl and adl to supported CPU model

In libpeci added rpl and adl CPU Model to supported CPU model

Tested:
Checked and confirmed by adding switch case for rpl and adl
in crashlog code

Signed-off-by: vijayabharathi shetty <vijayabharathix.shetty@intel.com>
Upstream-Status: Pending
---
 peci.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/peci.h b/peci.h
index 0e5884c..f212216 100644
--- a/peci.h
+++ b/peci.h
@@ -39,6 +39,8 @@ extern "C"
         emr = 0x000C06F0,
         gnr = 0x000A06D0,
         srf = 0x000A06F0,
+        rpl = 0x000B0670,
+        adl = 0x00090675,
     } CPUModel;
 
     // PECI Status Codes
-- 
2.17.1

